code/game/objects/structures/displaycase.dm

```diff
@@ -336,6 +350,10 @@
 		else
 			..()
 
+/obj/item/key/displaycase

```

Comment:
That's fine, I doubt it will ever be an issue, but it's just a small annoyance for me.